ros/src/aero_offboard_velocity/src/aero_offboard_velocity_node.cpp

```diff
@@ -0,0 +1,205 @@
+/**
+ * @file mavros_offboard_velocity_node.cpp
+ * @brief MAVROS Offboard Control example node, written with mavros version 0.14.2, px4 flight
+ * stack and tested in Gazebo SITL & jMAVSIM.
```

Comment:
You may remove **jMAVSIM** if you couldn't verify on it.

Reply:
Thanks for suggestion shakthi. But it works fine with jmavsim.